packages/manager/apps/web/client/app/domains/translations/Messages_de_DE.json

```diff
@@ -66,40 +66,6 @@
   "domains_configuration_dnssec_simple_activate_step1_question": "Möchten Sie wirklich DNSSEC für die Domain <span class=\"italic\">{{t0}}</span> aktivieren?",
   "domains_configuration_dnssec_simple_activate_success": "Ihre Anfrage zur Änderung des DNSSEC Zustands wurde erfolgreich durchgeführt.",
   "domains_configuration_dnssec_simple_activate_fail": "Bei der Anfrage für die Änderung des DNSSEC Zustands ist ein Fehler aufgetreten.",
-  "domains_newdnszone_order_title": "DNS Zone hinzufügen",
```

Comment:
Moved in https://github.com/ovh/manager/blob/master/packages/manager/apps/web/client/app/dns-zone/new/translations/Messages_fr_FR.json
https://github.com/ovh/manager/commit/16b5fd75bc2622574a0dadd8b613031fe5b7ef77#diff-276ebfd9368c1fa4cee67b0980a9f6efL69